libcentrifugo/metrics.go

```diff
@@ -109,6 +112,24 @@ type metricsRegistry struct {
 	mu sync.Mutex
 }
 
+func newMetricsHistogramRegistry() *hdrhistogram.HDRHistogramRegistry {
+	quantiles := []float64{50, 90, 99, 99.99}
+	var minValue int64 = 1        // as we record latencies in microseconds, min resolution 1mks
+	var maxValue int64 = 10000000 // as we record latencies in microseconds, max resolution 10s
+	numBuckets := 15              // histograms will be rotated every time we call UpdateSnapshot from outside.
+	sigfigs := 3
+	registry := hdrhistogram.NewHDRHistogramRegistry()
+	registry.Register(hdrhistogram.NewHDRHistogram("http_api", numBuckets, minValue, maxValue, sigfigs, quantiles))
+	registry.Register(hdrhistogram.NewHDRHistogram("client_api", numBuckets, minValue, maxValue, sigfigs, quantiles))
```

Comment:
Actually I don't fully understand how to implement separate metrics for all these different protocol request types.. because they all can be aggregated in array of commands in one response. Do you mean that we should insert histograms after we already unmarshaled JSON and already iterating over commands in request: for example wrap [this call](https://github.com/centrifugal/centrifugo/blob/master/libcentrifugo/client.go#L416) like this:
```
started := time.Now()
resp, err = c.connectCmd(&cmd)
c.app.metrics.histograms.RecordMicroseconds("client_connect", time.Now().Sub(started))
``` 
I think this will take nanoseconds btw.